03/3_is_array_overlap/main.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+
+int is_overlap(int nums1[], const size_t length1, int nums2[], const size_t length2);
+
+int main(void)
+{
+    int nums1[5] = {1, 2, 3, 4, 5};
+    int nums2[5] = {1, 2, 8, 9, 10};
+    int* nums3 = nums1;

```

Comment:
bad example. 

better example will be

int nums1[10];
int nums2[5];

int* num3 = nums1 + 2; // and length is like 5